x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

```diff
@@ -170,3 +213,27 @@ echo -n "VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw" | base64 <1>
 ----
 <1> Use `-n` so that the `echo` command doesn't print the trailing newline
 character
+
+The following example creates an API key with a restriction to the `search_application_query` workflow,
+which allows to call only <<search-application-search, Search Application Search API>>:
+
+[source,console]
+----
+POST /_security/api_key
+{
+  "name": "my-restricted-api-key",
+  "role_descriptors": {
+    "restricted-role": {
+      "index": [
+        {
+          "names": ["index-*"],
```

Comment:
Nit: I expect an actual search application query API key will be created with index read privileges for the search application name, e.g. "my-search-app", which is the alias for the backing indices instead of directly on the indices.

Reply:
Correct. I will update example.